stl/inc/xutility

```diff
@@ -3662,27 +3674,40 @@ _DestTy* copy(_ExPo&&, _FwdIt1 _First, _FwdIt1 _Last, _DestTy (&_Dest)[_DestSize
 #endif // _HAS_CXX17
 
 // FUNCTION TEMPLATE copy_n
+template <class _InIt, class _Diff, class _OutIt>
+constexpr _OutIt _Copy_n_core(_InIt _First, _Diff _Count, _OutIt _Dest) {

```

Comment:
`_core` seems really weird to me. Why not `_Copy_n_impl`?

Reply:
Because it's specifically the implementation that uses only the core language; the memmove implementation is also an implementation.

Reply:
It seems odd to me to name a library function on the basis that it doesn't call another library function - that fact doesn't seem significant to me. That said, I get that you want a name that contrasts with `_Copy_memmove`, and I have no better suggestion, so why not.